### PR TITLE
Upgrade version 0.27

### PR DESCRIPTION
**Pull request by ElviaBth**
Updates the module to the 27 version

### CODE REVIEW

File: .rubocop_ruby.yml [hunk 1]
```diff
@@ -13,12 +13,13 @@ AllCops:
...
   Exclude:
+    - "**/node_modules/**/*"
     - "**/vendor/**/*"
     - "development_app/**/*"
+    - "decidim_app-design/node_modules/**/*"
     - "spec/decidim_dummy_app/**/*"
     - "node_modules/**/*"
-    - "db/migrate/*"
-    - "db/schema.rb"
+    - "decidim-initiatives/lib/gem_overrides/origami/date.rb"
```
Comment (microstudi) [inline .rubocop_ruby.yml:22]:
```suggestion
```
No need for this line

File: .rubocop_ruby.yml [hunk 2]
```diff
@@ -13,12 +13,13 @@ AllCops:
     - "**/Gemfile"
     - "**/Rakefile"
   Exclude:
+    - "**/node_modules/**/*"
     - "**/vendor/**/*"
     - "development_app/**/*"
+    - "decidim_app-design/node_modules/**/*"
```
Comment (microstudi) [inline .rubocop_ruby.yml:19]:
```suggestion
```
No need for this one

File: .rubocop_ruby.yml [hunk 3]
```diff
@@ -453,6 +463,7 @@ Naming/FileName:
     - "**/Gemfile"
     - "**/Rakefile"
     - "**/*.gemspec"
+    - "decidim-dev/lib/decidim-dev.rb"
```
Comment (microstudi) [inline .rubocop_ruby.yml:466]:
```suggestion
```

File: .rubocop_ruby.yml [hunk 4]
```diff
@@ -1250,6 +1261,8 @@ RSpec/MultipleExpectations:
 
 RSpec/MultipleMemoizedHelpers:
   Max: 35
+  Exclude:
+    - decidim-assemblies/spec/forms/assembly_form_spec.rb
```
Comment (microstudi) [inline .rubocop_ruby.yml:1265]:
```suggestion
```
we can skip this one

File: package.json [hunk 5]
```diff
@@ -38,9 +38,9 @@
     "@babel/plugin-transform-runtime": "^7.19.6",
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
-    "@decidim/dev": "0.27.1",
-    "@decidim/eslint-config": "0.27.1",
-    "@decidim/stylelint-config": "0.27.1",
+    "@decidim/dev": "0.27.3",
+    "@decidim/eslint-config": "0.27.3",
+    "@decidim/stylelint-config": "0.27.3",
```
Comment (microstudi) [inline package.json:43]:
```suggestion
    "@decidim/dev": "0.27.4",
    "@decidim/eslint-config": "0.27.4",
    "@decidim/stylelint-config": "0.27.4",
```
we can upgrade to 27.4 (do `npm i`)

File: lib/decidim/newsletter_agenda/version.rb [hunk 6]
```diff
@@ -2,8 +2,8 @@
 
 module Decidim
   module NewsletterAgenda
-    VERSION = "1.0.0"
-    DECIDIM_VERSION = "0.26.5"
-    COMPAT_DECIDIM_VERSION = [">= 0.26.0", "< 0.27"].freeze
+    VERSION = "1.1.0"
+    DECIDIM_VERSION = "0.27.3"
```
Comment (microstudi) [inline lib/decidim/newsletter_agenda/version.rb:6]:
```suggestion
    DECIDIM_VERSION = "0.27.4"
```
27.4 is out, we can change this (do `bundle` too)

**Review by microstudi — CHANGES_REQUESTED** [review]
Nice job @ElviaBth 
I see that 27.4 is out, we should change it, then you'll see some comments that you can apply.
THanks!

File: .rubocop_ruby.yml (hunk 1 above)
Comment (ElviaBth) [reply .rubocop_ruby.yml:22]: Done. Thanks

File: .rubocop_ruby.yml (hunk 2 above)
Comment (ElviaBth) [reply .rubocop_ruby.yml:19]: Done

**Pull request merged**